SofaKernel/modules/sofa/frameworkextra/frameworkextra_test/core/TrackedData_test.cpp

```diff
@@ -301,11 +301,8 @@ struct DataTrackerEngine_test: public BaseTest
         core::DataTrackerEngine dataTracker;
         dataTracker.addInput(&testObject.myData); // several inputs can be added
         dataTracker.addOutput(&testObject2.myData); // several output can be added
-        dataTracker.setUpdateCallback( &DataTrackerEngine_test::myUpdate );
+        dataTracker.addCallback( &DataTrackerEngine_test::myUpdate );
         dataTracker.setDirtyValue();
```

Comment:
No test for the addInputs / outputs? to show off your mad c++11 skills ;)

Reply:
Well.. showing c++11 skills while c++20 is approaching...sound like downgrading.